.idea/JMRI.iml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
+  <component name="CheckStyle-IDEA-Module">
+    <option name="configuration">
+      <map />
+    </option>
+  </component>
```

Comment:
Is this OK? Just checking so that these changes doesn't slip by by misstake.

Reply:
Also in PR #9433